Translator/Extractor/JMSTranslatorBundle/AdminExtractor.php

```diff
@@ -22,7 +22,11 @@
 use Symfony\Component\HttpKernel\Log\LoggerInterface;
 use Symfony\Component\Translation\TranslatorInterface;
 
-class AdminExtractor implements ExtractorInterface, TranslatorInterface, SecurityHandlerInterface, LabelTranslatorStrategyInterface
+final class AdminExtractor implements
+    ExtractorInterface,
```

Comment:
Ok I'll restore it for now but this makes no sense at all. Being consistent with bad code is producing bad code.